packages/react-hotkeys-hook/src/test/useHotkeys.test.tsx

```diff
@@ -549,18 +549,86 @@ test('should trigger sequence with useKey', async () => {
 })
 
 test('should not trigger sequence without useKey', async () => {
-  const user = userEvent.setup()
+  const user = userEvent.setup({delay: 200})
   const callback = vi.fn()
 
   renderHook(() => useHotkeys('%>!', callback, { useKey: false }))
 
-  await user.keyboard(`{Shift>}{%}{/Shift}`)
-  vi.advanceTimersByTime(200)
-  await user.keyboard(`{Shift>}{!}{/Shift}`)
+  await user.keyboard(`{Shift>}{%}{/Shift}{Shift>}{!}{/Shift}`)
 
   expect(callback).toHaveBeenCalledTimes(0)
 })
 
+test('should work with different ways of referencing modifiers', async () => {
+  const user = userEvent.setup()
+  const ctrlCallback = vi.fn()
+  const capsCallback = vi.fn()
+
+  renderHook(() => useHotkeys(['ctrl', 'ctl', 'control'], ctrlCallback))
+  renderHook(() => useHotkeys(['capslock', 'caps'], capsCallback))
+  
+  await user.keyboard(`{Control}`)
+  expect(ctrlCallback).toHaveBeenCalledTimes(3)
+
+  await user.keyboard(`{CapsLock}`)
+  expect(capsCallback).toHaveBeenCalledTimes(2)
+})
+
+test('should trigger sequence with modifier keys', async () => {
+  const user = userEvent.setup({delay: 200})
+  const callback = vi.fn()
+
+  renderHook(() => useHotkeys('shift>alt>control>caps>ctrl', callback))
+
+  await user.keyboard(`{Shift}{Alt}{Control}{CapsLock}{Control}`)
+
+  expect(callback).toHaveBeenCalledTimes(1)
+})
+
+test('should trigger sequence with duplicate keys', async () => {
+  const user = userEvent.setup({delay: 200})
+  const callback = vi.fn()
+
+  renderHook(() => useHotkeys('ctrl>g>g>ctrl', callback))
+
+  await user.keyboard(`{Control}gg{Control}`)
+
+  expect(callback).toHaveBeenCalledTimes(1)
+})
+
+test('should trigger sequence with modifier keys with useKey', async () => {
+  const user = userEvent.setup()
+  const callback = vi.fn()
+
+  renderHook(() => useHotkeys('shift>alt>caps>ctrl', callback, { useKey: true }))
+
+  await user.keyboard(`{Shift}{Alt}{CapsLock}{Control}`)
+  
+  expect(callback).toHaveBeenCalledTimes(1)
+})
```

Comment:
I don't think I understand what use cases this behavior unlocks. You can just remove `useKey` to get the same effect.

The purpose of `useKey` is to listen to the produced key, no matter _how_ it got produced, so we don't care about the keyboard layout. Checking for modifiers when setting `useKey` to true will counter this idea.